### DISCUSSION TITLE
MediaDC

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1701] [Ideas] [answered]
For users who use their Nextcloud at home and have a lot of data on it, this app is a must have!
This app/container under Optional addons would be very great.

https://github.com/andrey18106/mediadc-docker-example
https://apps.nextcloud.com/apps/mediadc
![grafik](https://user-images.githubusercontent.com/90777102/211196504-799d0b4e-0477-4805-91e7-e82fe0f5a73a.png)

**Comment by szaimen** [comment, marked as answer]
Hi, IIRC, you should already be able to install the [mediadc app](https://apps.nextcloud.com/apps/mediadc) after installing the [cloud_py_api app](https://apps.nextcloud.com/apps/cloud_py_api).

**Reply by ghost** [reply]
Yes I have just discovered too, unfortunately the app does not work with it.

To reproduce:
1. Activate cloud_py_api app
2. activate MediaDC app
3. create rule to delete in Media DC app
4. wait
5. the status of the task remains on Pending

**Reply by szaimen** [reply]
I guess best is to report this to the project then...
